### PR TITLE
raft_cmd: support cf.

### PR DESCRIPTION
**Pull request by disksing**
@siddontang @BusyJay @ngaut

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by disksing** [comment]
PTAL @BusyJay @ngaut

File: proto/raft_cmdpb.proto [hunk 1]
```diff
@@ -23,16 +24,18 @@ message SeekResponse {
...
 }
 
 message PutResponse {
     
 }
 
 message DeleteRequest {
-    optional bytes key = 1;
+    optional bytes  key = 1;
+    optional string cf  = 2;
```
Comment (BusyJay) [inline proto/raft_cmdpb.proto]: Should cf be the first field?

Comment (disksing) [inline proto/raft_cmdpb.proto]: I think both are ok and append to the end is more compatibility friendly.

Comment (BusyJay) [inline proto/raft_cmdpb.proto]: @ngaut Do we need to consider compatibility now?

Comment (disksing) [inline proto/raft_cmdpb.proto]: @ngaut Any suggestions?

Comment (ngaut) [inline proto/raft_cmdpb.proto]: Not now, let's keep it pretty.

**Comment by disksing** [comment]
PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**